### ISSUE TITLE
[BUG] Using an invalid option results in a Lua error

### ISSUE THREAD

**Issue by britzl** [levno-710/Prometheus#41]
**Describe the bug**
When you provide an invalid option the parser in cli.lua chokes when trying to present an error message:

```
cli.lua:86: bad argument #2 to 'format' (value expected)
stack traceback:
	[C]: in function 'format'
	...q53d3qg84mb4gjdm5hk40000gn/T/3334399680767961943/cli.lua:86: in main chunk
	[C]: at 0x01042bcd60
```
(please ignore the weird path, I'm, running Prometheus from a temp folder)

The problem is that string.format() is used but no argument is provided:

https://github.com/levno-710/Prometheus/blob/master/src/cli.lua#L86

**Expected behavior**
Prometheus should present which option that is invalid.

**To Reproduce**

1. Call Prometheus with an invalid option:
2. Observe error

**Comment by levno-710** [comment]
Thanks for reporting this issue,
I am currently on vacation, so I cannot access my development PC, but i will fix this once i'm Home.

**Comment by britzl** [comment]
Closed by #43
